www/header.inc

```diff
@@ -397,6 +400,12 @@ function GetGrades($testInfo, $testRunResults, $firstRunResults)
     } elseif (isset($infoArray['lighthouse']) && $infoArray['lighthouse']) {
       $grades['lighthouse'] = array( 'class' => 'F', 'grade' => '?', 'description' => 'Lighthouse PWA Score');
     }
+
+    require_once('security_checks.php');

```

Comment:
Can you make the security checks optional based on a setting that defaults to false? Something like GetSetting('checkSecurity'). That way it won't be enabled by default on private instances unless they opt-into it (which is how we handle the other external integrations).

Reply:
Ahh yes. I wrapped that part of the code with `GetSetting('securityInsights')` in this new commit: https://github.com/WPO-Foundation/webpagetest/pull/1349/commits/c162b28cc212bcadae63cae1469470f132a51192

I assumed that this means I don't need to move the security tests in custom_metrics/ but let me know if these still need to be moved into a security/ subfolder.

Reply:
Yes, still need to move it, otherwise those custom metrics scripts will run automatically for all private instances, adding overhead for no reason. They can just get a .security or .sample after the .js instead of moving them to a folder.

Reply:
@pmeenan cool I'll rename them. After I add the suffix, do I need to do anything else in particular to read them though?